dashboard/app/helpers/survey_results_helper.rb

```diff
@@ -1,18 +1,18 @@
 module SurveyResultsHelper
-  DIVERSITY_SURVEY_ENABLED = false
+  DIVERSITY_SURVEY_ENABLED = true
   NPS_SURVEY_ENABLED = false
 
   def show_diversity_survey?(kind)
     return false unless SurveyResultsHelper::DIVERSITY_SURVEY_ENABLED
     return false unless current_user
+    return false unless country_us?
```

Comment:
I change the order of these slightly to do the cheaper checks (i.e. the ones that don't require a query) first.  (Also so I don't need to update the HomeController tests that count queries.  :-)